src/Debugger/Agent.php

```diff
@@ -47,6 +47,8 @@ class Agent
     use BatchDaemonTrait;
     use SysvTrait;
 
+    const DEFAULT_LOGPOINT_LOG_NAME = 'appengine.googleapis.com%2Frequest_log';
```

Comment:
I'm not sure this is the right default. Presumably other languages are doing this, but it seems wrong name to me.

Why appengine? Why request_log?

Reply:
The Debug UI is making this query for logs: 

```
resource.type="gae_app"
logName="projects/[PROJECT_ID]/logs/appengine.googleapis.com%2Frequest_log" 
resource.labels.module_id="[SERVICE]" 
resource.labels.version_id="[VERSION]" 
timestamp>="[TIMESTAMP]" 
severity>=DEBUG
```

Other languages do not actually seem to be setting this log name, so I'm investigating what's going on.

Reply:
So other languages are not populating the UI log panel by default. For ruby, the default logs go to a Stackdriver Logger called 'debugger_logpoints'. Python use the default system logger. Both allow you to specify your own logger implementation which we already do.

We can set the default log name to this App Engine specific one (which links the UI) if we dectect that we are on App Engine. Otherwise, we'll default to 'debugger_logpoints' like the ruby agent does.